requirements.txt

```diff
@@ -41,3 +41,4 @@ pytest-benchmark
 pytest-cov
 pytest-integration
 pytest-mock
+pyobjc
```

Comment:
Please explain why is the requirement under Testing dependencies. Also can you give example of what is fixed with it?